src/osd/ECTransaction.h

```diff
@@ -178,11 +177,11 @@ namespace ECTransaction {
 	       (!plan.to_read.at(i.first).empty() &&
 		!i.second.has_source()));
       });
-    plan.t = std::move(t);
     return plan;
   }
 
   void generate_transactions(
+    PGTransaction* _t,
```

Comment:
We assert non-null, probably better to pass as a reference?

Reply:
Added a new commit for that.